### PR TITLE
added li list which is populated by API, styled li list with CSS

### PR DESCRIPTION
**Pull request by bbrown949**
[Screen Recording 2023-04-25 at 6.24.29 PM.zip](https://github.com/bbrown949/ajax-project/files/11328067/Screen.Recording.2023-04-25.at.6.24.29.PM.zip)

### CODE REVIEW

File: css/layout.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+body {
+  background-image: url("../images/concert.png");
+  background-size: 950px;
+}
+
+main {
+  width: 100%;
+}
+
+@media screen and (width: 768px) {
```
Comment (shawnkost) [inline css/layout.css:17]: This should be `min-width`

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
+const $form = document.querySelector('form');
+const $homePage = document.querySelector('div[data-view="home-page"]');
+const $concertsPage = document.querySelector('div[data-view="concerts-page"]');
+// const $idTitle = document.getElementById('new-entry-edit-entry');
```
Comment (shawnkost) [inline js/main.js:4]: Delete all old code

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  xhr.addEventListener('load', function () {
+    for (let i = 0; i < xhr.response._embedded.events.length; i++) {
+      const concertEntry = xhr.response._embedded.events[i];
+      const concertEntryElement = renderConcerts(concertEntry);
+      $availableShows.appendChild(concertEntryElement);
+    }
+    // console.log('xhr response', xhr.response);
+  });
+  xhr.send();
+  // console.log(event.target.elements.zip.value);
```
Comment (shawnkost) [inline js/main.js:20]: Delete all old code

**Review by shawnkost — CHANGES_REQUESTED** [review]
Nicely done Bryan! A few changes requested here

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  // $row.setAttribute('class', 'row');
+  // $imgLine.appendChild($row);
+
+  // const $columnHalfOne = document.createElement('div');
+  // $columnHalfOne.setAttribute('class', 'column-half');
+  // $row.appendChild($columnHalfOne);
+
+  // const $columnHalfTwo = document.createElement('div');
+  // $columnHalfTwo.setAttribute('class', 'column-half');
+  // $row.appendChild($columnHalfTwo);
```
[the remark this answers is not in the document]
Comment (bbrown949) [reply js/main.js]: all changes implemented

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**